nuxeo-lambda-image-conversion/src/main/java/org/nuxeo/lambda/image/conversion/listener/LambdaResponseListener.java

```diff
@@ -58,10 +61,18 @@
     public void handleEvent(Event event) {
         EventContext ctx = event.getContext();
         Map<String, Serializable> params = (Map<String, Serializable>) ctx.getProperty(PARAMETERS_KEY);
+        if (params == null || params.isEmpty()) {
+            log.warn("Couldn't get parameters from the context");
```

Comment:
This (and others) is probably not useful as a WARN. It think it should be DEBUG only.